### PR TITLE
[ticket/14112] Add variable to index event for board statistics

### PR DESCRIPTION
**Pull request by svennd**
Variable added so we can access the board
statistics from an extension;

PHPBB3-14112

### CODE REVIEW

File: phpBB/index.php [hunk 1]
```diff
@@ -192,11 +191,15 @@
 *
 * @event core.index_modify_page_title
 * @var	string	page_title		Title of the index page
+* @var	array	board_statistics		Array with the board statistics, legend and birthday list
 * @since 3.1.0-a1
```
Comment (marc1706) [inline phpBB/index.php]: Missing ```@changed```

File: phpBB/index.php [hunk 2]
```diff
@@ -161,8 +161,7 @@
 	$db->sql_freeresult($result);
 }
 
-// Assign index specific vars
-$template->assign_vars(array(
+$board_statistics = array(
```
Comment (marc1706) [inline phpBB/index.php]: This is not just the board statistics but rather most of the index template data apart from the forums. Maybe just use a variable name like $template_data

**Comment by marc1706** [comment]
@svennd Are you going to take care of the mentioned issues? If not, please give us a heads up so we can take care of it.

**Comment by svennd** [comment]
@marc1706 I'm sorry I thought you could edit it as a patch, I am unfamilar with git, I'm willing to look into it tomorrow, but if its only a click on some button for you feel free.

**Comment by svennd** [comment]
I don't understand why it failed ?

**Comment by marc1706** [comment]
I'll take care of it ;)

**Pull request closed without merging**